### PR TITLE
Update security document

### PR DESCRIPTION
**Pull request by cas--**
- Add more details about 2FA.
- Increase minimum length to 16 chars.
- Expand details on password managers
- Update details on remote wipe services.
- Reword sharing customer details.

### CODE REVIEW

File: company/security.md [hunk 1]
```diff
@@ -1,53 +1,72 @@
...
-* All computers must be locked with a password when kept running and left somewhere, even if only for a few minutes. 
+* Every computer or device must have a login password or pin to secure access.
+* Use random passwords with more than 16 characters.
+* Do not use the same password more than once.
+* All devices must be locked with a password when kept running and left somewhere, even if only for a few minutes.
 
-### Web Application Passwords
+### Password manager
+
+Use a password manager (1Password, LastPass, KeePass, etc.) to store all other passwords so you do not need to remember them and the passwords can be more complex.
```
Comment (dz0ny) [inline company/security.md]: Enpass please it's only one that is multiplatform and has same features as 1p.

File: company/security.md [hunk 2]
```diff
@@ -1,53 +1,72 @@
...
+
+The passwords stored in a password manager should be longer randomly generated passwords with a combination of uppercase and lowercase letters, numbers, and symbols.
```
Comment (dz0ny) [inline company/security.md]: Trust password manager to generate those...

File: company/security.md [hunk 3]
```diff
@@ -1,53 +1,72 @@
...
+
+* Store passwords for company access to 1Password team account.
```
Comment (dz0ny) [inline company/security.md]: Use of https://chrome.google.com/webstore/detail/1password-x-%E2%80%93-password-ma/aeblfdkhhhdcdjpifhhbdiojplfjncoa is recommended over having copy of passwords or desktop apps.

File: company/security.md [hunk 4]
```diff
@@ -1,53 +1,72 @@
...
+
+### Two-factor authentication (2FA)
+
+Where ever possible enable 2FA (two-factor authentication) for logging into services.
+
+Using an OTP (one-time-password) authentication application, e.g. Google Authenticator, is the most secure option. Avoid using SMS authentication, unless it is the only option, as it is vulnerable to attacks.
```
Comment (dz0ny) [inline company/security.md]: One can also use hardware based module (FIDO) like Yubikey for example.

File: company/security.md [hunk 5]
```diff
@@ -1,53 +1,119 @@
...
 
-* Macbooks and iPhones have that built-in. You can trace the device and also do a complete remote wipe of it.
-* Linux laptops and Android phones get theft control with Preyproject.com.
 
 ## Syncing
 
-Syncing of data between the device and any other device/service needs to be performed in a secure manner. Syncing over USB or local Wireless network is fine. Syncing through Dropbox, iCloud or Google Sync is *not* allowed as that would mean that all data is stored unencrypted at Dropbox/Apple/Google. 
+Syncing of data between the device and any other device/service needs to be performed in a secure manner.
+
+Syncing over USB, local wired or wireless network that are trusted is fine. Rembember a coffee shop or other public wifi cannot be trusted.
```
Comment (zupo) [inline company/security.md]: `Rembember` typo

**Pull request merged**